src/coreclr/src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ReadyToRunCodegenNodeFactory.cs

```diff
@@ -134,13 +134,11 @@ public override int GetHashCode()
         private struct ReadOnlyDataBlobKey : IEquatable<ReadOnlyDataBlobKey>
```

Comment:
Rename this to `SettableReadOnlyDataBlobKey` as well.

Reply:
Thanks. Renamed too.